mobile-ui/src/main/java/org/sagebionetworks/research/mobile_ui/perform_task/PerformTaskFragment.java

```diff
@@ -79,7 +81,7 @@
 /**
  * A placeholder fragment containing a simple view.
  */
-public class PerformTaskFragment extends Fragment implements HasSupportFragmentInjector {
+public class PerformTaskFragment extends Fragment implements HasSupportFragmentInjector, Serializable {
```

Comment:
https://medium.com/@hamidgh/sending-objects-to-fragment-naive-question-is-it-sent-by-value-ddaaa19fa42d